docs/introduction.rst

```diff
@@ -0,0 +1,88 @@
+************
+Introduction
+************
+
+Overview
+========
+
+``Remme`` is a blockchain-based protocol used for issuing and management of X.509 client certificates to resolve issues
+related to cybersecurity, IoT connectivity, data integrity, digital copyright protection, transparency etc.
+
+``Remme core`` is built on Hyperledger Sawtooth platform, allowing to be flexible in language choice during the
+development process. ``Remme core`` supports JS, .NET, that’s why you to easily embed protocol in your project.
```

Comment:
Not true. It exposes the RPC API which is wrapped with a bunch of integration libraries (JS, .NET, etc)

Reply:
@eugene-babichenko, done.